src/components/SelectButtonGroup/SelectButton/SelectButton.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+
+interface Props {
+    /** the label to be displayed on this button */
+    children: string;
+    /** the value associated with this button */
+    value: string;
```

Comment:
I think value also can be number or boolean.
For example, FE retrieves some boolean (true/false) value from BE and renders this value like two switch box buttons disabled/enabled. In the current implementation, we need to convert it to string and then back to boolean.

Reply:
I double checked and the native HTML supports only string. So if we allow for other types, we would need to convert them for rendering and then reconvert them. The reconverting will be quite tricky as we have to guess what it was sent as. I think this is too much risk to add this flexibility to a UI component, and would rather keep this logic in the application.